geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/metadata/MetadataSerializer.scala

```diff
@@ -11,15 +11,15 @@ package org.locationtech.geomesa.index.metadata
 import java.nio.charset.StandardCharsets
 
 trait MetadataSerializer[T] {
-  def serialize(typeName: String, key: String, value: T): Array[Byte]
-  def deserialize(typeName: String, key: String, value: Array[Byte]): T
+  def serialize(typeName: String, value: T): Array[Byte]
```

Comment:
`key` was not actually used in any of these methods, so I removed it from the signature.